### DISCUSSION TITLE
School name on signup incorrect

### DISCUSSION THREAD

**Discussion by wmjomt** [community/community#143911] [GitHub Education]
### Hi everyone,

My students cannot sign up because the school name on the form is "jolietschools.com" instead of "jolietschools.org"
Who can correct this in the system?
<img width="600" alt="github-jolietschools" src="https://github.com/user-attachments/assets/7d6eb631-bd49-4e73-8370-257dcfb6b92f">

**Comment by davevad93** [comment]
Hi @wmjomt ,

In this case, I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/education), select `My selected school might have incorrect or incomplete information e.g. domains or campus location`, and explain the issue with your school's name registered on GitHub Education.

**Comment by wmjomt** [comment]
Thank you for your quick response

**Comment by diyorbekw** [comment]
Hello
